lib/ReactViews/Map/Panels/SharePanel/ShareUrl/ShareUrl.tsx

```diff
@@ -126,6 +126,7 @@ export const ShareUrl = forwardRef<
         id="share-url"
         rounded={rounded}
         onCopy={(text) =>
+          // eslint-disable-next-line react/prop-types
```

Comment:
for some reason, prop-types lint doesn't like forwardRef, suggestion to look into those after upgrade

Reply:
I've seen a codemod that removed forwardRef, so I believe forwardRef will be deprecated in React 19. But I think it makes sense to get to React 18 before worrying about React 19.